website/screens/components/grid/code/GridCodePage.tsx

```diff
@@ -0,0 +1,327 @@
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>templateColumns: string[]</td>
+          <td></td>
+          <td>
+            Sets the <Code>grid-template-columns</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>templateRows: string[]</td>
+          <td></td>
+          <td>
+            Sets the <Code>grid-template-rows</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-rows"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>as: keyof HTMLElementTagNameMap</td>
+          <td>
+            <Code>'div'</Code>
+          </td>
+          <td>Sets a custom HTML tag.</td>
+        </tr>
+        <tr>
+          <td>children: node</td>
+          <td></td>
+          <td>Custom content inside the grid container.</td>
+        </tr>
+      </DxcTable>
+    ),
+  },
+  {
+    title: "DxcGrid.GridItem",
+    content: (
+      <>
+        <DxcParagraph>
+          This compound component should be used when the decision to follow the
+          Grid layout pattern is assumed only for the parent container. If the
+          inner layout of a grid item does not make this same compromise and
+          only features props related to the parent container, then that grid
+          item should be wrapped with this component.
+        </DxcParagraph>
+        <DxcParagraph>
+          In case you are building a complex Grid layout with several nested
+          grid containers, <Code>DxcGrid.GridItem</Code> becomes very limited
+          and therefore, you will have to wrap the children with{" "}
+          <Code>DxcGrid</Code>.
+        </DxcParagraph>
+        <DxcParagraph>
+          So, to fulfil that scenario,{" "}
+          <strong>
+            all the <Code>DxcGrid.GridItem</Code>'s props are also available in{" "}
+            <Code>DxcGrid</Code>.
+          </strong>
+        </DxcParagraph>
+      </>
+    ),
+    subSections: [
+      {
+        title: "Props",
+        content: (
+          <DxcTable>
+            <tr>
+              <th>Name</th>
+              <th>Default</th>
+              <HeaderDescriptionCell>Description</HeaderDescriptionCell>
+            </tr>
+            <tr>
+              <td>areaName: string</td>
+              <td></td>
+              <td>
+                Sets the name of an item so that it can be referenced by a
+                template created with the <Code>grid-template-areas</Code>{" "}
+                property.
+              </td>
```

Comment:
How do you see to reference here the `grid-area`? https://developer.mozilla.org/en-US/docs/Web/CSS/grid-area

Reply:
Same as above.